Add support for PaladinPad, Arya pcb and move keyboards by KapCave into their own directory

nachie opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

Add support for PaladinPad PCBs.

Move all keyboards maintained by nachie into the KapCave directory. This makes sure all PCBs with the same vendor ID are in the same directory. Generated a unique vendor ID (hex for KC), made sure it wasn't being used at the time.

Update the files to match the current standards required for PRs removing old code and null functions.

Add in via keymaps

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

drashna commented:
Just a heads up, moving keybof folders around usually means that the PR needs to target develop (you can edit the title to change the target).  If you want to still target master, then it may be worth splitting up the PR into two PRs. 

Also, there are some merge conflicts here.

nachie commented:
Changed the target to develop.

Ill rebase locally to see if I can get rid of the merge conflicts

nachie commented:
The lint for the new paths passes, but for some reason its trying to lint paladin64 in the old path and that is failing (since it was moved). Not sure how to handle that.

nachie commented:
Resolved merged conflicts again.

zvecr commented:
In its current form, this PR cannot be accepted due to the submodule changes in `lib`, most likely from before the previous develop cycle merged. These will need to be resolved before the PR can move forward.

nachie commented:
> In its current form, this PR cannot be accepted due to the submodule changes in `lib`, most likely from before the previous develop cycle merged. These will need to be resolved before the PR can move forward.

Deleted the old branch, updated develop and made a new branch based off of that. This should now be rid of all the submodule changes.

drashna commented on keyboards/kapcave/gskt00/keymaps/default-poly/config.h:
```diff
@@ -0,0 +1,30 @@
+
+#define RGB_DI_PIN D0
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

drashna commented on keyboards/kapcave/paladinpad/rev1/config.h:
```diff
@@ -0,0 +1,35 @@
+#define RGB_DI_PIN D0
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

drashna commented on keyboards/kapcave/paladinpad/rev2/config.h:
```diff
@@ -0,0 +1,38 @@
+#define RGB_DI_PIN D0
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

drashna commented on keyboards/kapcave/paladinpad/rules.mk:
```diff
@@ -0,0 +1,22 @@
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= no		# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes		# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no		# Console for debug(+400)
+COMMAND_ENABLE ?= no    	# Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  	# Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes			# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= no  	# Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no			# PCB Supports this, enable if you add a buzzer
+RGBLIGHT_ENABLE ?= yes
```
```suggestion
BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no       # Mouse keys
EXTRAKEY_ENABLE = yes      # Audio control and System control
CONSOLE_ENABLE = no        # Console for debug
COMMAND_ENABLE = no        # Commands for debug and configuration
SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes          # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
AUDIO_ENABLE = no          # PCB Supports this, enable if you add a buzzer
RGBLIGHT_ENABLE = yes
```

nachie commented:
Thank you for the suggestions @drashna , I've accepted all. I didn't know the RGB animations could be broken out, I should be reading the core files more I guess :)

nachie commented:
Adding in another PCB that is slated for GB alongside a board in the coming months.

drashna commented on keyboards/kapcave/arya/rules.mk:
```diff
@@ -0,0 +1,30 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

drashna commented on keyboards/kapcave/arya/rules.mk:
```diff
@@ -0,0 +1,30 @@
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
```suggestion
```

fauxpark commented on keyboards/kapcave/paladinpad/info.json:
```diff
@@ -0,0 +1,84 @@
+  "keyboard_name": "PaladinPad", 
+  "url": "https://kapcave.com/products/paladinpad-pcb", 
+  "maintainer": "nachie", 
+  "width": 4, 
+  "height": 5, 
```
```suggestion
```

fauxpark commented on keyboards/kapcave/paladin64/rules.mk:
```diff
@@ -0,0 +1,25 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes		# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes		# Mouse keys
+EXTRAKEY_ENABLE = yes		# Audio control and System control
+CONSOLE_ENABLE = yes		# Console for debug
+COMMAND_ENABLE = no    		# Commands for debug and configuration
+SLEEP_LED_ENABLE = no  		# Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes			# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  		# Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+# Use these options if you add in trackpoint hardware
+#PS2_MOUSE_ENABLE = yes
+#PS2_USE_USART = yes
+#PS2_USE_INT = yes
```
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

fauxpark commented on keyboards/kapcave/paladin64/readme.md:
```diff
@@ -0,0 +1,24 @@
+Firmware for the Paladin64 ALPS64 PCB, with underglow and an optional
+trackpoint module.
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: Paladin64 ALPS PCB 
+Hardware Availability: [KapCave](https://kapcave.com/products/paladin64-pcb)
```
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: Paladin64 ALPS PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/paladin64-pcb)
```

fauxpark commented on keyboards/kapcave/paladin64/info.json:
```diff
@@ -1,7 +1,9 @@
+    "keyboard_name": "Paladin64",
+    "url": "https://kapcave.com/products/paladin64-pcb",
+    "maintainer": "nachie",
+    "width": 15,
+    "height": 5,
```
```suggestion
```

fauxpark commented on keyboards/kapcave/gskt00/readme.md:
```diff
@@ -0,0 +1,23 @@
+
+Firmware for the GSKT-00 PCB
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: GSKT-00 PCB
+Hardware Availability: [KapCave](https://kapcave.com/products/gskt-00-pcb-usb-c)
```
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: GSKT-00 PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/gskt-00-pcb-usb-c)
```

fauxpark commented on keyboards/kapcave/arya/rules.mk:
```diff
@@ -0,0 +1,29 @@
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Wildcard to allow APM32 MCU
+#DFU_SUFFIX_ARGS = -p FFFF -v FFFF
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes
+#WS2812_DRIVER = pwm
```
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
```

fauxpark commented on keyboards/kapcave/arya/readme.md:
```diff
@@ -0,0 +1,23 @@
+
+Firmware for the KapCave Arya Keyboard
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: Arya PCB
+Hardware Availability: [KapCave](https://kapcave.com/products/arya-pcb)
```
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: Arya PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/arya-pcb)
```

fauxpark commented on keyboards/kapcave/arya/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,63 @@
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /* QWERTY */
+	[_BASE] = LAYOUT_alice_split_bs(
+        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS, \
+        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,          \
+        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           \
+                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            \
+                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL           \
+    ),  
+
+    /* FUNCTION */
+	[_FN] = LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
+    ),
+
+	[_FN2] = LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
+    ),
+	
+	[_FN3] = LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
```
```suggestion
	[_BASE] = LAYOUT_alice_split_bs(
        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS,
        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL
    ),  

    /* FUNCTION */
	[_FN] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
    ),

	[_FN2] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
    ),
	
	[_FN3] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
```

fauxpark commented on keyboards/kapcave/arya/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    LAYOUT_alice_split_bs(
+        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS, \
+        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,          \
+        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           \
+                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            \
+                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL           \
+    ),
+    LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
```
```suggestion
    LAYOUT_alice_split_bs(
        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS,
        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL
    ),
    LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
```

fauxpark commented on keyboards/kapcave/arya/info.json:
```diff
@@ -0,0 +1,79 @@
+    "keyboard_name": "Arya", 
+    "url": "https://kapcave.com/products/arya", 
+    "maintainer": "nachie", 
+    "width": 20, 
+    "height": 14.0, 
```
```suggestion
```

fauxpark commented on keyboards/kapcave/paladinpad/paladinpad.h:
```diff
@@ -0,0 +1,121 @@
+    { K_1, K_2, K_3, K_ENT }, \
+    { K_0, K_DOT1, K_10, K_11 }  \
+}
+
+/*
+Remove this
+#define KEYMAP( \
+    K_NUM1, K_EQ1, K_SLSH1, K_STAR1, \
+    K_7, K_8, K_9, K_MINUS1, \
+    K_4, K_5, K_6, K_PLUS, \
+    K_1, K_2, K_3, K_11, \
+    K_0, K_10, K_DOT1, K_ENT \
+) { \
+    { K_NUM1, K_EQ1, K_SLSH1, K_STAR1 }, \
+    { K_7, K_8, K_9, K_MINUS1 }, \
+    { K_4, K_5, K_6, K_PLUS }, \
+    { K_1, K_2, K_3, K_ENT }, \
+    { K_0, K_DOT1, K_10, K_11 } \
+}
+*/
+
```
```suggestion
```
